src/lib/utils/paths.ts

```diff
@@ -0,0 +1,28 @@
+import * as Path from 'path';
+import { isArray } from 'util';
+import { Minimatch, IMinimatch } from 'minimatch';
+
+function convertToMinimatch(pattern: string): IMinimatch {
+    // Ensure uniform absolute path cross OS
+    // (POSIX would resolve c:/path to /path/to/repo/c:/path without this check)
+    if (Path.isAbsolute(pattern) && Path.sep === '/') {
+        pattern = pattern.replace(/^\w:/, '');
+    }
+
+    if (pattern[0] !== '*') {
```

Comment:
Should this be checking for `**`? If the path is not resolved for `*/test.js` then I think it won't work with `/project/base/*/test.js` which is what I would expect

Reply:
hm... yeah you might be right... I will correct it.